app.rb

```diff
@@ -2,8 +2,16 @@
 
 module FormsLab
   class App < Sinatra::Base
+    get '/' do
+      erb :root
+    end
 
-    # code other routes/actions here
+    get '/new' do
+      erb :'pirates/new'
+    end
 
+    post '/pirates' do
+      erb :'pirates/show'
+    end
```

Comment:
Your POST action should also contain logic to create a new instance of a pirate and its associated ships.